src/hooks/types.ts

```diff
@@ -7,6 +9,10 @@ export enum QueryKeys {
   ReadModularComplianceGetModules = "readModularComplianceGetModules",
   ReadHtsTokenTokenAddress = "readHtsTokenTokenAddress",
   ReadAccountTokens = "readAccountTokens",
+  ReadTokenDecimals = "readTokenDecimals",
+  ReadTokenIsAgent = "readTokenIsAgent",
+  ReadTokenOwner = "ReadTokenIsAgent",
```

Comment:
ReadTokenOwner = "ReadTokenIsAgent" => ReadTokenOwner = "readTokenOwner"
ReadIdentityRegistryInvestorCountry = "ReadIdentityRegistryInvestorCountry" => ReadIdentityRegistryInvestorCountry = "readIdentityRegistryInvestorCountry"
Correct me if it's conceived, but just a misspelling.